README.md

```diff
@@ -85,6 +85,101 @@ Your minified file need to be appended with a comment directive to your source m
 //# sourceMappingURL=http://example.com/path/to/your/sourcemap.map
 ```
 
+## Setup for ReactJs
+
+### Initialization
+
+1. In the root `index.html` file which contains the application bundle, load the library before the `</head>` tag.
+
+```html
+<head>
+<!-- All of your other HTML <head> content -->
+<script src="https://cdn.rawgit.com/GoogleCloudPlatform/stackdriver-errors-js/v0.0.4/dist/stackdriver-errors-concat.min.js"></script>
+</head>
+```
+
+2. You can capture all exceptions with the following logic in your entry point, typically `index.js` with the following code:
```

Comment:
1. and 2. are not React Specific, they are exactly what is already described in the set up instructions. I'd suggest to just say that for React, users need to follow the regular setup instructions.